.github/workflows/monitor-test-deploy.yml

```diff
@@ -54,7 +54,8 @@ jobs:
     runs-on: ubuntu-latest
     env:
       REGION: us-east-1
-      SERVICE_NAME: teleport-monitor-prod
+      SERVICE_NAME_RINKEBY: teleport-monitor-prod
```

Comment:
Shouldn't this have `rinkeby` in the name, like the other one has `kovan`?

Reply:
The service is already deployed and working including the DB 😬 
Both Kovan and Rinkeby are getting deprecated so this will change soon anyway